### PR TITLE
Update README.md and misc repo cleanup

### PR DESCRIPTION
**Pull request by msherif1234**
- update readme and add logo
- fix spelling errors detected by goreport card

Signed-off-by: Mohamed Mahmoud <mmahmoud@redhat.com>

**- What this PR does and why is it needed**

**- Special notes for reviewers**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
-# ingress-node-firewall
+<p align="center">
+  <img src="logo.jpg" alt="logo" width="25%"/>
+</p>
+
+# Ingress Node Firewall
 This is the Ingress node Firewall Operator, implementing [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) for deploying Ingress node firewall daemon on kubernetes cluster.
 It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/)
 which provides a reconcile function responsible for synchronizing resources untile the desired state is reached on the cluster
```
Comment (martinkennelly) [inline README.md:8]:
```suggestion
which provides a reconcile function responsible for synchronizing resources until the desired state is reached on the cluster
```

Comment (martinkennelly) [reply README.md:8]: Can you make this fix too? Id say run the whole doc through a spell checker as well.

Comment (msherif1234) [reply README.md]: I ran spell check for the following
```bash
aspell check README.md 
aspell check test/README.md
```

**Pull request merged**